kolibri/plugins/learn/assets/src/vue/content-page/index.vue

```diff
@@ -48,35 +47,29 @@
 
 <script>
 
+  const constants = require('../../state/constants');
+  const getters = require('../../state/getters');
+
   module.exports = {
-    created() {
-      this.fetchFullContent(this.id);
-    },
+    mixins: [constants], // makes constants available in $options
```

Comment:
Yeah, I see what you're saying - these things could be a props in that sense. But yeah, definitely on the same page that the `[constants]` mixin is ugly.